src/styles/blocks/default.scss

```diff
@@ -0,0 +1,9 @@
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Don't reset margin and paddings in * selector, you can reset it for elements you are using

Reply:
https://github.com/mate-academy/layout_product-cards/pull/3400/files#r1272522914
didn't fix it